cernopendata/modules/fixtures/cli.py

```diff
@@ -51,9 +51,22 @@ def fixtures():
 @fixtures.command()
 @click.option('--skip-files', is_flag=True, default=False,
               help='Skip loading of files')
+@click.option('files', '--file', '-f', multiple=True,
+              type=click.Path(exists=True),
+              help='Path to the file(s) to be loaded. If not provided, all'
+                   'files will be loaded')
+@click.option('--profile', is_flag=True,
+              help='Outputs cProfile stats to stdout.')
```

Comment:
Output instead of outputs; we use declarative language in help.  Could be simplified to "Output profiling information".